ament_flake8/ament_flake8/main.py

```diff
@@ -139,8 +139,10 @@ def generate_flake8_report(config_file, paths, excludes, max_line_length=None):
         return generate_flake8_report(config_file, paths, excludes, max_line_length)
 
     flake8_argv = []
-    flake8_argv.append('--config={0}'.format(config_file))
-    flake8_argv.append('--exclude={0}'.format(excludes))
+    if config_file is not None:
+        flake8_argv.append('--config={0}'.format(config_file))
+    if excludes is not None:
```

Comment:
Which setting makes it not possible? From what I've seen `'--foo --bar 1 2` will make `foo` an empty list if it's an `nargs='*'` option.

My concern was consistency with the v2.x implementation, since it [doesn't use the argument if it's `[]`](https://github.com/ament/ament_lint/blob/5c9c8acaa5f0137f4544857038652c589e506a32/ament_flake8/ament_flake8/legacy.py#L76). but after re-looking at it, the end result will still turn out the same as 2.x with this implementation, so I'm fine to leave it as is